packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -68,17 +69,24 @@ export const getPatientByDemo = async ({
     ) {
       return true;
     }
-    // If the IDs don't match, or none were provided, check the demo for a match
-    let demoMatch =
-      intersectionWith(splitName(patient.data.firstName), splitName(demo.firstName), isEqual)
-        .length > 0 &&
-      intersectionWith(splitName(patient.data.lastName), splitName(demo.lastName), isEqual).length >
-        0 &&
-      intersectionWith(patient.data.address, demo.address, isEqual).length > 0;
-    if (demoMatch && demo.contact && demo.contact.length > 0) {
-      demoMatch = intersectionWith(patient.data.contact, demo.contact, isEqual).length > 0;
+
+    // Basic Implementation with semi-arbitrary threshold
+    const jw_score = calculatePatientSimilarity(patient.data, demo);
+    if (jw_score >= 0.95) {
+      return patient;
     }
-    return demoMatch;
+
+    // If the IDs don't match, or none were provided, check the demo for a match
+    // let demoMatch =
+    //   intersectionWith(splitName(patient.data.firstName), splitName(demo.firstName), isEqual)
+    //     .length > 0 &&
+    //   intersectionWith(splitName(patient.data.lastName), splitName(demo.lastName), isEqual).length >
+    //     0 &&
+    //   intersectionWith(patient.data.address, demo.address, isEqual).length > 0;
+    // if (demoMatch && demo.contact && demo.contact.length > 0) {
+    //   demoMatch = intersectionWith(patient.data.contact, demo.contact, isEqual).length > 0;
+    // }
+    // return demoMatch;
```

Comment:
Any particular reason to keep this around? If not let's remove this and also update the comment on lines 60-61 about doing this on the DB, please, we will likely not do this ever considering this new approach.